Header header
string signal 
float32 x
float32 y
float32 z
string object_type
